web/client/utils/mapinfo/__tests__/wms-test.js

```diff
@@ -108,4 +110,38 @@ describe('mapinfo wms utils', () => {
             }
         );
     });
+    it('test intercept ogc error in wms if success', (done)=>{
+        mockAxios.onGet().reply(() => {
+            return [200, {
+                "type": "FeatureCollection",
+                "features": [],
+                "totalFeatures": "unknown",
+                "numberReturned": 0,
+                "timeStamp": "2023-09-22T08:50:30.808Z",
+                "crs": null
+            }];
```

Comment:
I think here you can return the body of an exception in json

try to get it by requesting  a get feature info with exception format in application/json  for a layer that does not exist maybe